tests/kernel/static_idt/src/static_idt.c

```diff
@@ -108,27 +109,19 @@ int idt_stub_test(void)
 	p_idt_entry = (struct segment_descriptor *)
 		      (_idt_base_address + (TEST_SOFT_INT << 3));
 	offset = (u32_t)(&int_stub);
-	if (DTE_OFFSET(p_idt_entry) != offset) {
-		TC_ERROR("Failed to find offset of int_stub (0x%x) at vector %d\n",
-			 offset, TEST_SOFT_INT);
-		return TC_FAIL;
-	}
+
+	 /**TESTPOINT: Check offset of int_stub */
+	zassert_equal(DTE_OFFSET(p_idt_entry), offset,
+			"Failed to find offset of int_stub at vector");
 
 	/* Check for the exception stub */
 	p_idt_entry = (struct segment_descriptor *)
 		      (_idt_base_address + (IV_DIVIDE_ERROR << 3));
 	offset = (u32_t)(&_EXCEPTION_STUB_NAME(exc_divide_error_handler, 0));
-	if (DTE_OFFSET(p_idt_entry) != offset) {
-		TC_ERROR("Failed to find offset of exc stub (0x%x) at vector %d\n",
-			 offset, IV_DIVIDE_ERROR);
-		return TC_FAIL;
-	}
 
-	/*
-	 * If the other fields are wrong, the system will crash when the exception
-	 * and software interrupt are triggered so we don't check them.
-	 */
-	return TC_PASS;
+	/**TESTPOINT: Check offset of exc stub */
+	zassert_equal(DTE_OFFSET(p_idt_entry), offset,
+			"Failed to find offset of exc stub at vector");
```

Comment:
you are omitting lots of information and those messages are not accurate anymore without the numbers, we probably need to add support for formatting to zassert to be able to print out those numbers

Reply:
@nashif 
I added following PR for that
https://github.com/zephyrproject-rtos/zephyr/pull/4399